src/styles/base/_forms.scss

```diff
@@ -11,6 +11,22 @@ select {
   display: block;
   font-family: $base-font-family;
   font-size: $base-font-size;
+  height: $forms-height;
+  border: $base-border;
+
+    &:hover {
+      border-color: shade($dusty-gray, 10%);
+    }
+
+    &:focus {
+        border-color: $action-color;
+        outline: none;
+    }
+}
+::-webkit-input-placeholder{
```

Comment:
We don't put vendor prefixes here, they'll be added later through gulp (but I still need to implement that so if you need it leave it there for now)